src/main/frontend/components/settings.cljs

```diff
@@ -667,14 +667,20 @@
   []
   [:div.panel-wrap
    [:div.text-sm.my-4
+    (ui/admonition
```

Comment:
Can we put a warning or at least a "use at your own risk" at the beginning about using Sync and auto git commit together? We discourage it in the [Sync FAQ](https://docs.logseq.com/#/page/logseq%20sync/block/faq) so we should try to be consistent